dev/com.ibm.ws.microprofile.config_fat/test-applications/cdiConfig.jar/src/com/ibm/ws/microprofile/appConfig/cdi/web/SysPropServlet.java

```diff
@@ -12,26 +12,52 @@
 
 import java.io.IOException;
 import java.io.PrintWriter;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
 
+import javax.inject.Inject;
 import javax.servlet.ServletException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.ibm.ws.microprofile.appConfig.cdi.beans.SessionScopedConfigFieldInjectionBean;
+
 @SuppressWarnings("serial")
 @WebServlet("/system")
 public class SysPropServlet extends HttpServlet {
```

Comment:
This seems to be doing a whole bunch more than it used to...

Reply:
Did you collapse `SessionScopedFieldTestServlet` into this class? Can it be removed?

Reply:
ah yes